travel_platform/skyscanner_data/skyscanner_data.py

```diff
@@ -15,57 +15,41 @@
 inbound_date = config.get('api_skyscanner', 'inbound_date')
 flights_cache_service = FlightsCache(api_key)
 
+#FIXME: why my *args not work?
```

Comment:
So, who and what should fix in scope of this `FIXME`?

I also don't understand.